apps/src/sites/studio/pages/teacher_dashboard/show.js

```diff
@@ -31,6 +31,7 @@ import currentUser, {
 } from '@cdo/apps/templates/currentUserRedux';
 import {setValidScripts} from '../../../../redux/unitSelectionRedux';
 import locales, {setLocaleCode} from '@cdo/apps/redux/localesRedux';
+import {setStudentsForCurrentSection} from '../../../../templates/teacherDashboard/teacherSectionsRedux';
```

Comment:
Could we just add `setStudentsForCurrentSection` to the list of functions imported from teacherSectionsRedux above?

Reply:
Whoops - good catch, will move